NuRadioReco/framework/parameters.py

```diff
@@ -64,13 +67,19 @@ class channelParameters(Enum):
     bandwidth_NuRadioMC_simulation = 20  #: the integrated channel response (=bandwidth for signal chains without amplification) used in the MC simulation
```

Comment:
Regarding:
```
    SNR = 5  #: an dictionary of various signal-to-noise ratio definitions
```
I know that has nothing to do with your PR but you have a good overview what is in there now. Do you think you could add a much more extensive description of what is in this dict?